doc/gazebo_simulation/gazebo_simulation.rst

```diff
@@ -0,0 +1,351 @@
+            - panda_joint1
+            - panda_joint2
+            - panda_joint3
+            - panda_joint4
+            - panda_joint5
+            - panda_joint6
+            - panda_joint7
+        - name: panda_hand_controller
+            action_ns: follow_joint_trajectory
+            type: FollowJointTrajectory
+            default: true
+            joints:
+            - panda_finger_joint1
+            - panda_finger_joint2
+
+
+7. Adjust auto-generated ros_controllers.launch
+-----------------------------------------------
+
+Fill the :code:`args` in line 9 with:
+
+.. code-block:: xml
+
+    joint_state_controller panda_hand_controller panda_arm_controller
+
+-------------------------------
+
+At last purely Gazebo way of using the panda robot is ready! In order to be able to control the robot via a simpler
+GUI, install `rqt_joint_trajectory_controller <http://wiki.ros.org/rqt_joint_trajectory_controller>`_.
+
+In terminal-1:
+
+.. code-block:: xml
+
+    roslaunch panda_moveit_config gazebo.launch
+
+In terminal-2:
+
+.. code-block:: xml
+
+    rosrun rqt_joint_trajectory_controller rqt_joint_trajectory_controller
+
+
+.. figure:: pure-gazebo.gif
+   :width: 700px
+
+   Panda arm control in Gazebo simulation.
+
+If you happen to find all these steps too tedious (you cannot be blamed for that), just clone `the franka_ros fork <https://github.com/tahsinkose/franka_ros>`_, that is created 
+particularly for this tutorial with the final versions of the files mentioned in the previous steps.
+The changes made thus far in auto-generated :code:`panda_moveit_config` package are `in this repository <https://github.com/tahsinkose/panda_moveit_gazebo_config>`_. 
+At the end, both repositories will have the updated and directly usable versions.
+
+-------------------------------
+
+Now it is time to integrate MoveIt to this work. Open :code:`panda_moveit_config/launch/demo_gazebo.launch` file 
+and replace line 61 with:
+
+.. code-block:: xml
+
+    <arg name="rviz_config" value="$(find panda_moveit_config)/launch/moveit.rviz"/>
+
+This will allow us to use juicy Motion Planning display of MoveIt in Rviz. There is a final minor issue in `demo_gazebo.launch` file. Remove
+line 31 from that file, which contains unused :code:`urdf_path` argument. After that, launch:
+
+.. code-block:: xml
+
+    roslaunch panda_moveit_config demo_gazebo.launch
+
+.. figure:: moveit-gazebo.gif
+   :width: 700px
+
+   Panda arm controlled via MoveIt in Gazebo simulation.
+
+
+We have successfully integrated MoveIt and Gazebo ultimately. MoveIt Setup Assistant already does
+many work under the hood, but it still misses some parts to provide a proper Gazebo integration. After following
+this tutorial you should be able to reproduce this locally for any robot. In case you don't want to be 
+bothered with all the details, `franka_ros <https://github.com/tahsinkose/franka_ros>`_ and `panda_moveit_config <https://github.com/tahsinkose/panda_moveit_gazebo_config>`_
```

Comment:
The link to `https://github.com/tahsinkose/panda_moveit_gazebo_config` is broken--maybe you meant `https://github.com/tahsinkose/panda_moveit_config`? Also, please say which branches should be used, or link directly to them. Looks like the `simulation` branch on `franka_ros` and the `melodic-devel` branch on `panda_moveit_config`, right?

Reply:
> The link to https://github.com/tahsinkose/panda_moveit_gazebo_config is broken--maybe you meant https://github.com/tahsinkose/panda_moveit_config?

Yes, thanks for notification :+1: 

> Looks like the simulation branch on franka_ros and the melodic-devel branch on panda_moveit_config, right

Yes to both.

Reply:
```suggestion
bothered with all the details, `franka_ros <https://github.com/tahsinkose/franka_ros/tree/simulation>`_ and `panda_moveit_config <https://github.com/tahsinkose/panda_moveit_config/tree/melodic-devel>`_
```

Reply:
Thanks!